contracts/test/forkMainnet/L2Deposit/Setup.t.sol

```diff
@@ -129,4 +131,50 @@ contract TestL2Deposit is StrategySharedSetup {
         (uint8 v, bytes32 r, bytes32 s) = vm.sign(privateKey, EIP712SignDigest);
         return abi.encodePacked(r, s, v, bytes32(0), uint8(SignatureValidator.SignatureType.EIP712));
     }
+
+    function _createSpenderPermitFromL2Deposit(L2DepositLibEIP712.Deposit memory _deposit) internal view returns (SpenderLibEIP712.SpendWithPermit memory) {
+        SpenderLibEIP712.SpendWithPermit memory spendWithPermit = SpenderLibEIP712.SpendWithPermit(
+            _deposit.l1TokenAddr, // tokenAddr
+            address(l2Deposit), // requester
+            _deposit.sender, // user
+            address(l2Deposit), // recipient
+            _deposit.amount, // amount
+            getEIP712Hash(l2Deposit.EIP712_DOMAIN_SEPARATOR(), L2DepositLibEIP712._getDepositHash(DEFAULT_DEPOSIT)), // actionHash
+            uint64(_deposit.expiry) // expiry
+        );
+
+        return spendWithPermit;
+    }
+
+    function _signSpendWithPermit(uint256 _privateKey, SpenderLibEIP712.SpendWithPermit memory _spendWithPermit) internal returns (bytes memory) {
```

Comment:
Should be replaced by `signSpendWithPermit` in #114 .

Reply:
fixed!